src/main/scala/chisel3/iotesters/CommandEditor.scala

```diff
@@ -0,0 +1,85 @@
+// See LICENSE for license details.
+
+package chisel3.iotesters
+
+import java.io.File
+
+import scala.io.Source
+import scala.util.matching.Regex
+
+/**
+  * This function applies a last chance method of making final alteration of the vcs command line.
+  * Alterations are made from a text file containing ed style regex substitutions
+  * s/regex-pattern/substitution/ or more generally
+  * s<<separator>>regex-pattern<<separator>>substitution<<separator>>
+  * if the file begins with the line verbose, the substitution parsing and operation will be logged to stdout
+  *
+  * @param editCommands  a seq of strings containing edits
+  * @param messagePrefix   stick this in front of any warnings or errors
+  * @return
+  */
+//noinspection MatchToPartialFunction
+class CommandEditor(val editCommands: Seq[String], messagePrefix: String) {
+  def apply(command: String): String = {
+    var verbose = false
+    def show(s: => String): Unit = if(verbose) println(s)
+    var currentCommandLine = command
+
+    editCommands.foreach { //noinspection MatchToPartialFunction
+      line =>
+      line match {
+        case CommandEditor.RegexPrefixPattern(separator) =>
+          val editPatternString = s"""s$separator([^$separator]*)$separator([^$separator]*)$separator.*"""
+          val EditPattern = editPatternString.r
+          line match {
+            case EditPattern(pattern, substitution) =>
+
+              val newCommandLine = pattern.r.replaceAllIn(currentCommandLine, substitution)
+              if(newCommandLine != currentCommandLine) {
+                show(s"""$messagePrefix applying "$pattern" => "$substitution" yields "$newCommandLine" """)
+              }
+              else {
+                show(s"""$messagePrefix applying "$pattern" => "$substitution"  did not change string""")
+              }
+              currentCommandLine = newCommandLine
+            case _ =>
+              show(s"""$messagePrefix no match for "$editPatternString" on command "$currentCommandLine" """)
+          }
+        case CommandEditor.Verbose() =>
+          verbose = true
+          show(s"""$messagePrefix applying edits to "$currentCommandLine" """)
+        case _ =>
+          show(s"""vcs-command-edit ignoring edit command "$line" """)
+      }
+    }
+    currentCommandLine
```

Comment:
Where do you think the right place to put that message would be, in the command line help?
I don't quite understand the last comment about unchecked flags